ansible_collections/arista/avd/molecule/eos_designs-twodc-5stage-clos/intended/configs/DC1.POD1.LEAF2A.cfg

```diff
@@ -113,6 +115,7 @@ interface Port-Channel18
    no shutdown
    switchport
    switchport access vlan 110
+   switchport mode access
```

Comment:
We need to consider if we want this change of config.

Reply:
This because we are not specifically excluding `access`  in template, unlike how it was done with the deprecated keys.
In EOS it is default and does not show this command in running-config

Reply:
Tracking this decision separately, so we can move on.